### PR TITLE
nsxiv: init at 27.1

### PR DESCRIPTION
**Pull request by tdillenburg**
###### Motivation for this change
[sxiv](https://github.com/muennich/sxiv/) repository has been archived. Development's been moved to [nsxiv](https://github.com/nsxiv/nsxiv). Add nsxiv package to nixpkgs.

The package is essentially the same as [sxiv](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/graphics/sxiv/default.nix). I just ommited the maintainer because I was not sure it should be the same.

###### Things done

- Built on platform(s)
  - [X] NixOS 21.11
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -12845,5 +12845,11 @@
     github = "mbprtpmnr";
     githubId = 88109321;
   };
+  dillenburg = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:12848]: Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Also please squash all package commits.

File: pkgs/top-level/aliases.nix [hunk 2]
```diff
@@ -853,6 +853,7 @@ mapAliases ({
   solr_8 = solr; # added 2021-01-30
   spectral = neochat; # added 2020-12-27
   sundials_3 = throw "sundials_3 was removed in 2020-02. outdated and no longer needed";
+  sxiv = nsxiv; # added 2021-10-17
```
Comment (rapenne-s) [inline pkgs/top-level/aliases.nix:856]: I disagree with this change, you can't replace `sxiv` with `nsxiv` transparently if the new package doesn't provide the same binary name.

People will install `sxiv`, the installation process will work but user won't find `sxiv`, this is very confusing for users.

File: pkgs/top-level/all-packages.nix [hunk 3]
```diff
@@ -27659,6 +27659,8 @@ with pkgs;
 
   sxiv = callPackage ../applications/graphics/sxiv { };
```
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/top-level/all-packages.nix:27660]: Also we would need to delete `applications/graphics/sxiv`.

**Comment by nagy** [comment]
With https://github.com/NixOS/nixpkgs/pull/161854 being merged, this can be closed.
